holotrack/Vision/FaceTracking.cs

```diff
@@ -9,9 +9,9 @@ namespace holotrack.Vision
     /// <summary>
     /// A class that implements face targeting using DLib.
     /// </summary>
-    public class FaceTracking
+    public static class FaceTracking
     {
-        public static FaceRecognition faceRecognition;
+        internal static FaceRecognition FaceRecognition { get; set; } = FaceRecognition.Create(".models");
```

Comment:
Instead of this being everywhere, we could do something like:
`Game.cs`
```csharp
private void load()
{
    // This is ran only once as the game is loaded.
    dependency.CacheAs(FaceRecognition.Create("models"))
}
```
so later we can use it anywhere: `AnyComponent.cs`
```csharp
[Resolved]
private FaceRecognition faceRecognition { get; set; } // value gets populated because of the Resolved attribute
```

Reply:
Probably the best idea as whenever this class is called this would create another instance.

Reply:
Doesn't `FaceRecognition.Create` create an instance of `FaceRecognition` populated with data from the directory/files it read? When would we need another instance of `FaceRecognition` then?

Reply:
I don't think so, we only use it on two classes, so we're not really reusing it in any way.